wis2node/topic_hierarchy.py

```diff
@@ -38,6 +38,8 @@ def __init__(self, path: str) -> None:
             self.dotpath = path
             self.dirpath = path.replace('.', '/')
 
+        self.collection_name = self.dotpath.lower()
```

Comment:
is the above for the Elasticsearch indexing?  This should be in the ES API backend plugin.

Reply:
I had it in ES at first but the backend plugin just takes in collection_id. Putting it inside topic_hierarchy.py felt like it made the most sense from a canonical reference to the correct name for the collection index.

Reply:
fwiw...

short version:   I think a  list is a better way to represent a topic hierarchy in python.

justification:
just to mention that the topic separator varies by message queueing protocol.  In AMQP, the separator is dot or period "."  like in python class hierarchies.  in MQTT, the topic separator is a forward slash "/" like in non-windows paths.  obviously when writing a path, it will have "/" ... but publishing to AMQP it will need "."

In python logic, I tend to use a list for the topic hierarchy, and apply the correct separator only when actually publishing in a specific protocol....